### PR TITLE
Fix/err in windows installtion

### PR DESCRIPTION
**Pull request by SiddheshKukade**
Fixes the error in windows installtion.
mentioned in #438 

###  Scripts working fine in Windows:
image

## Scripts working fine in Linux 
image

### CODE REVIEW

**Comment by SiddheshKukade** [comment]
@DeviaVir  Please review

File: client/package.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
   "author": "Nadav Ivgi",
   "license": "MIT",
   "scripts": {
-    "postinstall": "../node_modules/.bin/patch-package",
-    "dist": "../node_modules/.bin/babel -d dist src"
+    "postinstall": "cd ../node_modules/.bin &&  patch-package",
+    "dist": "cd ../ && cd node_modules && cd .bin || babel -d dist src"
```
Comment (DeviaVir) [inline client/package.json:8]: What is this `||` (OR) condition based on? Did you mean `&&`?

**Comment by DeviaVir** [comment]
Can you post a screenshot of the error you get on Windows before this change?

File: client/package.json (hunk 1 above)
Comment (SiddheshKukade) [reply client/package.json:8]: Right I should be &&.
I'll fix it

**Comment by SiddheshKukade** [comment]
@DeviaVir Here it is:  It says ".. "not found as a command.
image

File: client/package.json [hunk 2]
```diff
@@ -4,8 +4,8 @@
   "author": "Nadav Ivgi",
   "license": "MIT",
   "scripts": {
-    "postinstall": "../node_modules/.bin/patch-package",
-    "dist": "../node_modules/.bin/babel -d dist src"
+    "postinstall": "cd ../node_modules/.bin &&  patch-package",
+    "dist": "cd ../ && cd node_modules && cd .bin && babel -d dist src"
```
Comment (DeviaVir) [inline client/package.json:8]:
```suggestion
    "dist": "cd ../node_modules/.bin && babel -d dist src"
```

File: client/package.json [hunk 3]
```diff
@@ -4,8 +4,8 @@
   "author": "Nadav Ivgi",
   "license": "MIT",
   "scripts": {
-    "postinstall": "../node_modules/.bin/patch-package",
-    "dist": "../node_modules/.bin/babel -d dist src"
+    "postinstall": "cd ../node_modules/.bin &&  patch-package",
```
Comment (DeviaVir) [inline client/package.json:7]:
```suggestion
    "postinstall": "cd ../node_modules/.bin && ./patch-package",
```

Comment (SiddheshKukade) [reply client/package.json:7]: Actually sir, `./` was causing problem in windows.
With the errors like :`.` `is not recognized as a command` .
and not adding the `./`was working fine in linux.

**Review by DeviaVir — APPROVED** [review]

**Pull request merged**